app/assets/stylesheets/middleware_topology.css

```diff
@@ -0,0 +1,9 @@
+.middleware .kube-topology g.MiddlewareDatasource text.glyph {
+    font-size: 20px;
+    fill: #81B0C8;
+}
+
+.middleware .kube-topology g text.attached-label {
```

Comment:
I'm definitely in favor of doing a common "display names" thingy for all topologies in separate PR.